admin/cleanup.py

```diff
@@ -26,20 +26,12 @@
 from flocker.testtools.cluster_utils import MARKER
 
 
-def get_creation_time(node):
-    """
-    Get the creation time of a libcloud node.
-
-    Rackspace and EC2 store the information in different metadeta.
-
-    :return: The creation time, if available.
-    :rtype: datetime or None
-    """
-    date_string = node.extra.get("created", node.extra.get("launch_time"))
-    if date_string is None:
-        return None
-    else:
-        return parse_date(date_string)
+# By default only nodes with names beginning with these prefixes will be
+# considered for cleanup.
+DEFAULT_NODE_NAME_PREFIXES = (
+    'acceptance-test-',
+    'client-test-'
```

Comment:
Probably, but the script will currently only cleanup AWS instances in the region where Jenkins runs acceptance tests.